mixer/tools/codegen/pkg/bootstrapgen/generator.go

```diff
@@ -107,6 +107,13 @@ func (g *Generator) Generate(fdsFiles map[string]string) error {
 			"getAliasType": func(goType string) string {
 				return aliasTypes[goType]
 			},
+			"isAliasTypeSkipIp": func(goType string) bool {
```

Comment:
Not sure why you changed the name? It looks like we're not creating an overload, just changing the behavior a bit.

Perhaps a comment would be good.